### PR TITLE
FFWEB-2084: add feedback-campaigns blocks to search result page

### PR DESCRIPTION
**Pull request by paulfcdd**
- Fixes 
Web Components Oxid Module Feedback Text implementation for Bergspetzl

- Description: 
Web Components Oxid Module Feedback Text implementation for Bergspetzl

- Tested with Oxid EShop editions/versions: 
6.2.4
- Tested with PHP versions: 
7.3

### CODE REVIEW

File: src/views/frontend/blocks/search/result.tpl [hunk 1]
```diff
@@ -1,4 +1,13 @@
 <div class="boxwrapper">
+
+    [{assign var="oConfig" value=$oViewConf->getConfig()}]
+
+    [{if $oConfig->getConfigParam("ffCampaigns")}]
+        [{block name="ff_campaign_feedback_top"}]
```
Comment (a-laurowski) [inline src/views/frontend/blocks/search/result.tpl:6]: making this two blocks having same name `ff_campaign_feedback_top` will not give us benefit of overwriting only one.  In fact  i'm not sure if this is not an incorrect setup to have two blocks with the same name

Comment (paulfcdd) [reply src/views/frontend/blocks/search/result.tpl:6]: I believe it's incorrect setup, I forgot to rename second block, now it fixed

**Comment by a-laurowski** [comment]
I'd say that placing the campaign above the text "Your search for: produced 60 results" looks better than if it was placed above, what do you think?

image

**Comment by a-laurowski** [comment]
image
In case of below search result i'd say the one above pagination is better

File: src/views/frontend/blocks/search/result.tpl [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+            [{include file="ff/campaign/feedbacktext.tpl" label="above products"}]
```
Comment (a-laurowski) [inline src/views/frontend/blocks/search/result.tpl:7]:
```suggestion
            [{include file="ff/campaign/feedbacktext.tpl" label="above search result"}]
```

File: src/views/frontend/blocks/search/result.tpl [hunk 3]
```diff
@@ -19,4 +28,9 @@
             [{include file="ff/paging.tpl"}]
         </div>
     </div>
+    [{if $oConfig->getConfigParam("ffCampaigns")}]
+        [{block name="ff_campaign_feedback_bottom"}]
+            [{include file="ff/campaign/feedbacktext.tpl" label="belove products"}]
```
Comment (a-laurowski) [inline src/views/frontend/blocks/search/result.tpl:33]:
```suggestion
            [{include file="ff/campaign/feedbacktext.tpl" label="below search result"}]
```

**Review by a-laurowski — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,9 @@
 ### Changed
 - Upgrade Web Components version to v4.0.3
 
+### Added
+- Add feedback-campaigns blocks to search result page
```
Comment (a-laurowski) [inline CHANGELOG.md:13]: lets use real component name
```suggestion
- Add `ff-campaign-feedbacktext` elements to search result page
```

**Review by a-laurowski — CHANGES_REQUESTED** [review]

**Review by a-laurowski — APPROVED** [review]

**Pull request merged**